rasgointelligence/RasgoTransforms
# Preparing for pypi packaging

This PR prepares our repo to become an open-source package on pypi.
Changes:
- move all package files under folders: /rasgoudt/rasgoudt
- add a function in the /main.py file to serve these to python consumers
- add pypi files: DESC, setup.py, version
- add docker resources to assist in deployment

Updates complements of @alexwolf22:
- edit Rasgo internal publishing scripts to use new folder paths

TODO:
- test
- add code to auto-increment package version on deploy
- confirm package name
- publish package
